homeassistant/components/openrgb/select.py

```diff
@@ -0,0 +1,95 @@
+"""Select platform for OpenRGB integration."""
+
+from __future__ import annotations
+
+from homeassistant.components.select import SelectEntity
+from homeassistant.core import HomeAssistant, callback
+from homeassistant.exceptions import HomeAssistantError
+from homeassistant.helpers.device_registry import DeviceInfo
+from homeassistant.helpers.entity_platform import AddConfigEntryEntitiesCallback
+from homeassistant.helpers.update_coordinator import CoordinatorEntity
+
+from .const import CONNECTION_ERRORS, DOMAIN
+from .coordinator import OpenRGBConfigEntry, OpenRGBCoordinator
+
+PARALLEL_UPDATES = 0
+
+
+async def async_setup_entry(
+    hass: HomeAssistant,
+    config_entry: OpenRGBConfigEntry,
+    async_add_entities: AddConfigEntryEntitiesCallback,
+) -> None:
+    """Set up the OpenRGB select platform."""
+    coordinator = config_entry.runtime_data
+    async_add_entities([OpenRGBProfileSelect(coordinator, config_entry)])
+
+
+class OpenRGBProfileSelect(CoordinatorEntity[OpenRGBCoordinator], SelectEntity):
+    """Representation of an OpenRGB profile select entity."""
+
+    _attr_translation_key = "profile"
+    _attr_has_entity_name = True
+    _attr_current_option: str | None = None
```

Comment:
You define the current option (and sets the default), but I never see it set in the code base?